src/index.html

```diff
@@ -3,12 +3,46 @@
   <head>
     <meta charset="UTF-8">
     <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+          content="width=device-width,
+          user-scalable=no,
+          initial-scale=1.0,
+          maximum-scale=1.0,
+          minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="big__header">
+      <a href="#home" class="logo"><img src="images/logo.png" alt="Logo"></a>
+      <nav class="big__nav">
+        <ul>
+          <li><a class="nav__link is-active">Apple</a></li>
+          <li><a class="nav__link">Samsung</a></li>
+          <li><a class="nav__link">Smartphones</a></li>
+          <li><a class="nav__link" data-qa="hover">Laptops &amp; Computers</a></li>
+          <li><a class="nav__link">Gadgets</a></li>
+          <li><a class="nav__link">Tablets</a></li>
+          <li><a class="nav__link">Photo</a></li>
+          <li><a class="nav__link">Video</a></li>
+        </ul>
+      </nav>
+    </header>
+    <header class="small__header">
```

Comment:
remove this header, it should be the only one, but adaptive (fit on 1024px and 1200px )